doc.go

```diff
@@ -238,5 +238,70 @@ as well:
 	url, err := r.Get("article").URL("subdomain", "news",
 	                                 "category", "technology",
 	                                 "id", "42")
+
+Since **vX.Y.Z**, mux supports the addition of middlewares to a [Router](https://godoc.org/github.com/gorilla/mux#Router), which are executed if a
```

Comment:
Update to align with the README

(Note to self: use tooling to make this automatic...)

Reply:
I removed the version thingie. I'm not sure if its an important enough information to be in the readme. Feel free to re-add it if you think it is.

Reply:
Good move. I think it's fair to assume that the documentation reflects `HEAD`.

Reply:
Hi there. I can see that the online documentation still has the `*vX.Y.Z*`. Also the link `[Router](...)` hasn't been converted.

<img width="858" alt="screen shot 2018-01-18 at 11 58 40" src="https://user-images.githubusercontent.com/2451004/35094913-1371d94c-fc47-11e7-89de-73623797fcba.png">

Reply:
Yikes, I totally missed that one :/
I can fix it and create a new PR if you want, but maybe you can edit it quicker @elithrar

Reply:
I’ll take a look today. The website doesn’t do Markdown, which is a bit of a pain.

Reply:
https://github.com/gorilla/mux/pull/333

Reply:
Cool, thanks guys :)